.envrc

```diff
@@ -0,0 +1,2 @@
+export IMAGE_REGISTRY="localhost:5000"
```

Comment:
Having this file here is going to clobber potential valid settings for anyone working in this repo using direnv

Reply:
How do you propose we should go about handling this then? Maybe just adding to the README as a prereq to what is needed to run dobi?